### PR TITLE
Add DISCOURSE_REDIS_DB to make it configurable

### PR DESCRIPTION
**Pull request by nenych**
### Description of the change

Sometimes we need to configure Redis to use a database different than 0, this PR makes it configurable.

### Benefits

We can set REDIS_DB during setup.

### Possible drawbacks

No drawbacks

### Applicable issues

No issues

### Additional information

This feature is important to everyone who uses more than 1 DB per Redis setup.

### CODE REVIEW

File: bitnami/discourse/3/debian-12/rootfs/opt/bitnami/scripts/discourse-env.sh [hunk 1]
```diff
@@ -172,6 +174,8 @@ DISCOURSE_REDIS_PORT_NUMBER="${DISCOURSE_REDIS_PORT_NUMBER:-"${REDIS_PORT_NUMBER
 export DISCOURSE_REDIS_PORT_NUMBER="${DISCOURSE_REDIS_PORT_NUMBER:-6379}"
 DISCOURSE_REDIS_PASSWORD="${DISCOURSE_REDIS_PASSWORD:-"${REDIS_PASSWORD:-}"}"
 export DISCOURSE_REDIS_PASSWORD="${DISCOURSE_REDIS_PASSWORD:-}"
+DISCOURSE_REDIS_DB="${DISCOURSE_REDIS_DB:-"${REDIS_DB:-}"}"
+export DISCOURSE_REDIS_DB="${DISCOURSE_REDIS_DB:-}"
```
Comment (gongomgra) [inline bitnami/discourse/3/debian-12/rootfs/opt/bitnami/scripts/discourse-env.sh:178]: Can you force the default value here to be `0` and also add a new validation into `libdiscourse.sh`? There is a common `is_positive_int()` method that can be of help.

**Review by gongomgra — CHANGES_REQUESTED** [review]
Thanks for your contribution!

File: bitnami/discourse/3/debian-12/rootfs/opt/bitnami/scripts/discourse-env.sh (hunk 1 above)
Comment (nenych) [reply bitnami/discourse/3/debian-12/rootfs/opt/bitnami/scripts/discourse-env.sh:178]: Sure, will do it.

File: bitnami/discourse/3/debian-12/docker-compose.yml
Comment (gongomgra) [inline bitnami/discourse/3/debian-12/docker-compose.yml:1]: Please undo your testing changes

**Review by gongomgra — CHANGES_REQUESTED** [review]

File: bitnami/discourse/3/debian-12/rootfs/opt/bitnami/scripts/discourse-env.sh (hunk 1 above)
Comment (nenych) [reply bitnami/discourse/3/debian-12/rootfs/opt/bitnami/scripts/discourse-env.sh]: Done

File: bitnami/discourse/3/debian-12/docker-compose.yml
Comment (nenych) [reply bitnami/discourse/3/debian-12/docker-compose.yml:1]: Fixed, sorry.

**Review by gongomgra — APPROVED** [review]
Thanks for your contribution!

**Pull request merged**